modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -4,25 +4,31 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="modifying-an-existing-ingress-controller_{context}"]
-= Modifying an existing Ingress Controller
+= Modifying an existing Ingress Controller for manual DNS management
 
 As a cluster administrator, you can modify an existing Ingress Controller to manually manage the DNS record lifecycle.
 
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* Log in as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (oc).
+* You are logged in as a user with cluster-admin privileges.
 
 .Procedure
 
-. Modify the chosen `IngressController` to set `dnsManagementPolicy`:
-
+. Modify the chosen IngressController to set the dnsManagementPolicy: 
 +
 [source,terminal]
 ----
-SCOPE=$(oc -n openshift-ingress-operator get ingresscontroller <name> -o=jsonpath="{.status.endpointPublishingStrategy.loadBalancer.scope}")
+$ SCOPE=$(oc -n openshift-ingress-operator get ingresscontroller <name> -o=jsonpath="{.status.endpointPublishingStrategy.loadBalancer.scope}")
 
-oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"dnsManagementPolicy":"Unmanaged", "scope":"${SCOPE}"}}}}'
+$ oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"dnsManagementPolicy":"Unmanaged", "scope":"${SCOPE}"}}}}'
```

Comment:
This command will not work as expected because `$SCOPE` variable will not be expanded due to the single quotes. The quotes have to changed to double quotes and all double quotes in between have be escaped:
```
$ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch="{\"spec\":{\"endpointPublishingStrategy\":{\"type\":\"LoadBalancerService\",\"loadBalancer\":{\"dnsManagementPolicy\":\"Unmanaged\", \"scope\":\"${SCOPE}\"}}}}"
ingresscontroller.operator.openshift.io/default patched
```